lib/langchain/assistants/prompts/tools_prompt.yaml

```diff
@@ -0,0 +1,16 @@
+_type: prompt
+input_variables:
+  - tools
+template: |
+  You have access to the following tools:
+
+  {tools}
+
+  In order to use a tool, you can use <tool></tool> and <tool_input></tool_input> tags. The tool output will be provided to you in a follow-up chat message.
+
+  For example, if you have a tool called "google_search" that could run a Google search, in order to search for the weather in SF you would respond: <tool>google_search</tool><tool_input>weather in SF</tool_input>
+
+  user: What is the current weather in Boston, MA?
+  assistant: <tool>google_search</tool><tool_input>Boston, MA</tool_input>
```

Comment:
Minor point, but I think this example should be `<tool_input>weather in Boston, MA</tool_input>`, since the google_search tool wouldn't know what you're querying about. (It's correct in line 11.)
Another minor point is, if you add the `weather` tool, the llm might get confused about which tool to call. Maybe use an example that should always go to google_search.

Reply:
Openai recommends priming an assistant with faked examples, so I think <example_tool> might work well.
Also a few examples with  "role": "system", "name":"example_user".